crates/parcel_filesystem/src/js_delegate_file_system.rs

```diff
@@ -1,115 +1,28 @@
 use std::path::Path;
 use std::path::PathBuf;
-use std::rc::Rc;
-
-use napi::bindgen_prelude::FromNapiValue;
-use napi::Env;
-use napi::JsObject;
-use napi::JsUnknown;
-use parcel_napi_helpers::call_method;
 
 use crate::FileSystem;
 
-/// An implementation of `FileSystem` that delegates calls to a `JsObject`.
-///
-/// This is going to be very slow at runtime due to the overhead of converting
-/// between rust and JS types.
-pub struct JSDelegateFileSystem {
-  env: Rc<Env>,
-  js_delegate: JsObject,
-}
-
-impl JSDelegateFileSystem {
-  pub fn new(env: Rc<Env>, js_delegate: JsObject) -> Self {
-    Self { env, js_delegate }
-  }
-}
-
-fn path_from_js(result: JsUnknown) -> napi::Result<PathBuf> {
-  let result_string = result.coerce_to_string()?;
-  let result_string = result_string.into_utf8()?.as_str()?.to_string();
-  Ok(PathBuf::from(result_string))
-}
-
-// Convert arbitrary errors to io errors. This is wrong; the `FileSystem` trait should use
-// `anyhow::Result`
-fn run_with_errors<T>(block: impl FnOnce() -> anyhow::Result<T>) -> Result<T, std::io::Error> {
-  let result = block();
-  result.map_err(|err| std::io::Error::new(std::io::ErrorKind::Other, err.to_string()))
-}
+pub struct JSDelegateFileSystem {}
```

Comment:
Why do this? Better to just delete it

Reply:
The JS requests depend on it and I didn't want to make the PR too big

Reply:
Updated